### DISCUSSION TITLE
riscv time

### DISCUSSION THREAD

**Discussion by grigohas** [ggml-org/llama.cpp#10601] [Q&A]
Hello, can someone explain why the loading time of a model in llama-cli in riscv with vector enabled is less than without vector extsnsion?

**Comment by slaren** [comment]
It shouldn't make a significant difference. You may be seeing other effects such as the model already being cached in the system cache.

**Comment by grigohas** [comment]
there is a a difference of 119,821.82 ms . I want to ask if there is a different quantization when the rvv is enabled
